Podfile.lock

```diff
@@ -7,6 +7,6 @@ DEPENDENCIES:
 SPEC CHECKSUMS:
   OpenTok: 9f0909068d6ed69acffa77a4610b47924926f79e
 
-PODFILE CHECKSUM: d4b63399d3fef5486044af412f4a623ab30c4aa3
+PODFILE CHECKSUM: 3dc4397fcacc9a6d8f8ab5d446cc97ad3f96e91c
```

Comment:
The reason is to force cocoapods to use that specific version of the SDK. 
I've seen in the past that when you do a `pod install` it installs a random version of the SDK because the local repo is not updated. Moreover, sometimes when cocoapods version changes, it breaks the build so having the Podfile.lock there guarantees that you need to use a specific version of cocoapods.